particula/aerosol_dynamics/parcel_test.py

```diff
@@ -18,21 +18,88 @@
     charge=1,
 )
 
+standard_environment = environment.Environment(
+    temperature=298 * u.K,
+    pressure=101325 * u.Pa,
+)
+
 # create a parcel
-simple_parcel = parcel.Parcel('simple', temperature=300, pressure=101325)
+simple_parcel = parcel.Parcel('simple', standard_environment)
 
 # add single particle to parcel
 simple_parcel.add_particle(small_particle)
 simple_parcel.add_particle(large_particle)
-simple_parcel.create_and_add_particle('org1', 50e-9)
+simple_parcel.create_and_add_particle('org1', 500*u.nm)
 # add multiple particles to parcel
-simple_parcel.create_and_add_list_of_particle('org2', [1e-9, 2e-9, 3e-9])
+simple_parcel.create_and_add_list_of_particle('org2', [1e-9, 2e-9, 3e-9] * u.m)
+simple_parcel.create_and_add_list_of_particle(
+    'org2', [1e-9, 2e-9, 3e-9] * u.m,
+    [1.8, 1, 1] * u.kg / u.m ** 3, [1, 0, 2] * u.dimensionless
+)
 
 
 def test_getters():
     """
     Test that the getters work by confirming particle creation
     """
+    assert len(simple_parcel.particle_classes()) == 9
+    assert len(simple_parcel.particle_mass()) == 9
+    assert len(simple_parcel.particle_radius()) == 9
+    assert len(simple_parcel.particle_charge()) == 9
+
+
+def test_particle_mass_units():
+    '''
+    Test that the mass of the particles is returned in kg
+    '''
+    assert sum([i.mass().check('kg')
+                for i in simple_parcel.particle_classes()]) == 9
+
+
+def test_particle_radius_units():
+    '''
+    Test that the radius of the particles is returned in m
+    '''
+    assert sum([i.check('m') for i in simple_parcel.particle_radius()]) == 9
+
+
+def test_particle_density_units():
+    '''
+    Test that the density of the particles is returned in kg/m^3
+    '''
+    assert sum([i.check(u.kg / u.m ** 3)
+                for i in simple_parcel.particle_density()]) == 9
+
+
+def test_particle_charge_units():
+    '''
+    Test that the charge of the particles is returned in dimensionless
+    '''
+    assert sum([i.check(u.dimensionless)
+                for i in simple_parcel.particle_charge()]) == 9
+
+
+def test_particle_knudsen_number():
+    '''
+    Test that the knudsen number is returned in dimensionless
+    '''
+    assert sum([i.check(u.dimensionless)
+                for i in simple_parcel.particle_knudsen_number()]) == 9
+
+
+def test_remove_particle():
+    '''
+    Test that the remove particle method works
+    '''
+    simple_parcel.remove_particle([0])
+    assert len(simple_parcel.particle_classes()) == 8
+    simple_parcel.remove_particle([2, 4])
     assert len(simple_parcel.particle_classes()) == 6
-    assert len(simple_parcel.particle_mass()) == 6
-    assert len(simple_parcel.particle_radius()) == 6
+
+
+def test_remove_all_particles():
+    '''
+    Test that the remove all particles method works
+    '''
+    simple_parcel.remove_all_particles()
+    assert len(simple_parcel.particle_classes()) == 0
```

Comment:
Good job on the 0 from flake8!!!!

```
  # stop the build if there are Python syntax errors or undefined names
  flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
  # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
  flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.7.12/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.7.12/x64/lib
0
0
```

Reply:
ha, I check both pylint and flake8 now :)